contrib/format-httpd/README.md

```diff
@@ -0,0 +1,32 @@
+# Web Server Log Format Plugin (HTTPD)
+This plugin enables Drill to read and query httpd (Apache Web Server) and nginx logs natively. This plugin uses the work by [Niels Basjes](https://github.com/nielsbasjes) which is available here: https://github.com/nielsbasjes/logparser.
+
+## Configuration
+There are three fields which you will need to configure in order for Drill to read web server logs which are:
+* **`logFormat`**:  The log format string is the format string found in your web server configuration.
```

Comment:
Add something like: `If you have multiple sufficiently different logFormats then you can add all of them in this single parameter separated by a newline. The parser will automatically select the first matching format.`

Reply:
Fixed